app/views/layouts/_flash.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if flash[:error] %>
-  <div class="flash error row mx-0 p-3 align-items-center">
+  <div class="alert alert-danger row mx-0 mb-0 p-3 rounded-0 align-items-center">
```

Comment:
I'm happy to see the move from custom CSS to use the built-in bootstrap alert panels and colours. There's a couple of things here that need addressing though:

* If the `flash` CSS is no longer required elsewhere, then it should be removed from `common.scss`
* Large numbers of tests are failing, since they are looking for the `flash` and `error` css classes. The tests need to be updated.

Reply:
> If the `flash` CSS is no longer required elsewhere, then it should be removed from `common.scss`

There are quite a few tests which require the old classes and they were passing on the first test. Would it be OK if I update all instances of the current `flash` classes to `alert` (as well as their tests) or it would it be better to do it in a separate PR?

> Large numbers of tests are failing, since they are looking for the `flash` and `error` css classes. The tests need to be updated.

I've updated them. There should be only one failing about a `title` on the "Send message" page.

Reply:
You are right. I was confused since a few tests succeeded even though I didn't update them. They are now fixed.